### PR TITLE
Time-of-Use Tariff Controller UI: remove unused channel subscription

### PR DESCRIPTION
**Pull request by venu-sagar**
->  Removed unused channels.
->  Made channels to subscribe based on factory id.

### CODE REVIEW

File: ui/src/app/edge/history/timeofusetariffdischarge/widget.component.ts [hunk 1]
```diff
@@ -63,12 +60,21 @@ export class TimeOfUseTariffDischargeWidgetComponent extends AbstractHistoryWidg
         });
     }
 
-    protected getChannelAddresses(edge: Edge, config: EdgeConfig): Promise<ChannelAddress[]> {
+    protected override getChannelAddresses(edge: Edge, config: EdgeConfig): Promise<ChannelAddress[]> {
+        const result: ChannelAddress[] = [];
+        // Component was not initialized before, so had to initialize here.
+        this.service.getConfig().then(config => {
+            this.component = config.getComponent(this.componentId);
+        }).then(() => {
```
Comment (lukasrgr) [inline ui/src/app/edge/history/timeofusetariffdischarge/widget.component.ts:68]: chaining it 2 times is unecessary

**Review by lukasrgr — CHANGES_REQUESTED** [review]

File: ui/src/app/edge/history/timeofusetariffdischarge/widget.component.ts (hunk 1 above)
Comment (lukasrgr) [reply ui/src/app/edge/history/timeofusetariffdischarge/widget.component.ts:68]: can it happen that no channels are sent with the promise at the end of the function?

**Review by lukasrgr — APPROVED** [review]

**Review by sfeilmeier — APPROVED** [review]
Ok 👍

@venu-sagar: Please remember, after merge+release of #2238 we have to drop the `DelayedTime`.

**Comment by venu-sagar** [comment]
> Ok 👍
> 
> @venu-sagar: Please remember, after merge+release of #2238 we have to drop the `DelayedTime`.

We still need it. However, not based on the Factory ID, but based on the control mode selected.
So, control Mode 'CHARGE_CONSUMPTION' -> 'chargedTime', 'DELAY_DISCHARGE' -> 'delayedTime'

**Pull request merged**